### PR TITLE
Enemy ai patrol state

### PR DESCRIPTION
**Pull request by CrazyTobi**

### CODE REVIEW

**Review by AMacLeod99 — APPROVED** [review]

File: Source/SteveSloth/Public/MyEnemyPatrolState.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+UCLASS(Blueprintable)
+class STEVESLOTH_API UMyEnemyPatrolState : public UMyEnemyBaseState
+{
+	GENERATED_BODY()
+
+private:
+	AMyEnemyBaseClass* PMyself;
+	USkeletalMeshComponent* PMyMesh;
+	USplineComponent* PMySpline;
+	float Speed;
```
Comment (Saretzky) [inline Source/SteveSloth/Public/MyEnemyPatrolState.h:31]: get movement speed from base state

**Review by Saretzky — CHANGES_REQUESTED** [review]

File: Source/SteveSloth/Public/MyEnemyPatrolState.h (hunk 1 above)
Comment (CrazyTobi) [reply Source/SteveSloth/Public/MyEnemyPatrolState.h:31]: I do set the variable on line 17 in the cpp to the movement speed from base state

**Review by Saretzky — APPROVED** [review]
looks good

**Review by Jeff-Moreau — APPROVED** [review]
Looks Good

**Pull request merged**